spec/features/publish_a_document_spec.rb

```diff
@@ -31,5 +31,6 @@ def and_i_confirm_the_publishing
   def then_i_see_the_publish_succeeded
     expect(@request).to have_been_requested
     expect(page).to have_content "Publish successful"
+    expect(@document.reload.publication_state).to eql("live_on_govuk")
```

Comment:
Can we check this using the content?

Reply:
Also, is there a reason to use eql here? I think we mostly use eq elsewhere.

Reply:
Not sure this is important, we use both on GOV.UK. If you feel strongly you could make a PR to govuk-lint to enforce a default.